rafter/README.md

```diff
@@ -1,10 +1,10 @@
-# Asset Store
+# Rafter
```

Comment:
Please update the `Installation` procedure  in point 4:
Change:
`kubectl get assets.assetstore.kyma-project.io webpage -o jsonpath='{.status.phase}'`
to:
`kubectl get assets.rafter.kyma-project.io webpage -o jsonpath='{.status.phase}'`